core/templates/dev/head/pages/exploration_player/AudioControlsDirective.js

```diff
@@ -0,0 +1,120 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for a set of audio controls for a specific
+ * audio translation in the learner view.
+ */
+
+oppia.directive('audioControls', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        getAudioTranslations: '&audioTranslations'
+      },
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/exploration_player/' +
+        'audio_controls_directive.html'),
+      controller: [
+        '$scope', 'AudioTranslationManagerService', 'AudioPlayerService',
+        function(
+            $scope, AudioTranslationManagerService, AudioPlayerService) {
+          var filenameOfLastStartedAudio = null;
+          var directiveId = Math.random().toString(36).substr(2, 10);
+
+          $scope.AUDIO_SETTINGS_BUTTON_IMAGE_URL = (
+            UrlInterpolationService.getStaticImageUrl(
+              '/icons/settings.svg'));
+
+          var PLAY_AUDIO_BUTTON_IMAGE_URL = (
```

Comment:
Naming: use

IMAGE_URL_AUDIO_SETTINGS
IMAGE_URL_PLAY_AUDIO
IMAGE_URL_PAUSE_AUDIO
IMAGE_URL_REWIND_AUDIO

(basically, start with the type. That way it's easier to see that everything is in a "group of related things".)